pages/api/rollup/[cid].js

```diff
@@ -0,0 +1,24 @@
+// import { retrieveFilesData } from '../../../util/web3Storage'
```

Comment:
Good addition, although I may go with `pages/api/skills/[id]/rollup` approach to understand we are expecting a skill, unless we decide otherwise. One more item to cover during our pair with the team this week.